src/index.js

```diff
@@ -91,7 +91,7 @@ const loadConfig = (requiredKeys, file, actionType) => {
     .then(res => res.json())
     .then(config => {
       requiredKeys.forEach(key => {
-        if (!config[key]) throw new Error(`${key} is not set in ${file}. Coral will not work correctly.`);
+        if (typeof config[key] === "undefined") throw new Error(`${key} is not set in ${file}. Coral will not work correctly.`);
```

Comment:
the key could be `null` or empty string here (like in a directly copied `config.sample.json`). If we check directly for `undefined` people are going to get unexpected behavior.